<launch>
  <include file="$(find usb_cam)/launch/usb_cam-yuyv.launch" />


  <node name="opentld_node" pkg="tld_tracker" type="opentld_node" respawn="false" output="screen" >
    <param name="image" value="/usb_cam/image_raw" />
    <param name="target" value="trackingObject" />
  </node>


 </launch>
